network/datadog_checks/network/network.py

```diff
@@ -269,24 +271,28 @@ def _is_collect_cx_state_runnable(self, proc_location):
         if self._collect_cx_state is False:
             return False
 
-        if proc_location != "/proc":
+        if proc_location not in self.STANDARD_PROC_LOCATION:
             self.warning("Cannot collect connection state: currently with a custom /proc path: %s" % proc_location)
```

Comment:
From the user perspective, I would like to see all the allowed proc location here.
Because let's say I'm setupping this in a container, I couldn't know I can run it on `/host/proc` and of course I don't want to override my container `/proc` by the host's one.

Or: could you run the collection of connection state if the proc is mounted as `/host/proc` ?

Reply:
Do you mind explaining the rational for appending `/1` ?

```python
        if Platform.is_containerized() and proc_location != "/proc":
            proc_location = "%s/1" % proc_location

```

Trying to understand what are the circumstances when some files are under `/proc/1` and when it's not.

---

```
docker run -it -v /proc/:/host/proc/:ro datadog/agent:latest bash
root@28ff65c93ff7:/# ls -l /host/proc/sys/net/netfilter/nf_conntrack_max
-rw-r--r-- 1 root root 0 Jul 18 16:36 /host/proc/sys/net/netfilter/nf_conntrack_max

root@28ff65c93ff7:/# ls -l /host/proc/1/sys/net/netfilter/nf_conntrack_max
ls: cannot access '/host/proc/1/sys/net/netfilter/nf_conntrack_max': No such file or directory

root@28ff65c93ff7:/# ls /host/proc/1
attr	   auxv    clear_refs  comm ....
```
Seems `/host/proc/1` exist, but some files like `nf_conntrack_max` are under `/host/proc`, not `/host/proc/1`.

The issue seems to be that we are using `/host/proc` instead of `/host/proc/1` here https://github.com/DataDog/integrations-core/blob/0497a14bbd1c0601f230c3e9f4a81aa2eed5ec7b/network/datadog_checks/network/network.py#L424

Reply:
the proc is indeed being mounted as `/host/proc` but the network check modules looks at `/host/proc/1` path if the agent is running in container environment.